.github/workflows/test_open_data_assets.yaml

```diff
@@ -0,0 +1,58 @@
+name: test-open-data-assets
+
+on:
+  schedule:
+    # Every third day of the month at 9am CST (3pm UTC). This schedule
+    # makes sense since most of our assets are configured to refresh on the
+    # first day of the month. Some assets have a biweekly schedule such
+    # that they also refresh in the middle of the month, but we ignore that
+    # in order to make scheduling simpler.
+    - cron: '0 15 3 * *'
+
+jobs:
+  test-open-data-assets:
+    runs-on: ubuntu-latest
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint
+    # so that we can authenticate with AWS
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup dbt
+        uses: ./.github/actions/setup_dbt
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
+
+      - name: Install Python requirements
+        run: pip install requests
```

Comment:
It would probably be better practice to install from a requirements file, but given that there's only one dependency and we're using a super minimal and stable version of its API (`requests.get()`), I figure an inline install is probably good enough for now.

Reply:
**question:** It [looks like](https://github.com/dbt-labs/dbt-core/blob/main/core/setup.py#L91) `requests` is a dependency of `dbt-core`, so shouldn't this be handled by `setup_dbt`?

Reply:
Good catch, I removed this block in [dd01ef6](https://github.com/ccao-data/data-architecture/pull/156/commits/dd01ef6ee608379335167b9249f2261bc3b26c49)!